pkg/runner/cluster_k8s.go

```diff
@@ -72,12 +72,6 @@ const (
 	NetworkInitialisationFailed     = "network initialisation failed"
 )
 
-var (
```

Comment:
This was supposed to be addressed in https://github.com/testground/testground/pull/1008